control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -310,6 +310,7 @@ func ReconcileKubeAPIServerDeployment(deployment *appsv1.Deployment,
 
 	if auditWebhookRef != nil {
 		applyKASAuditWebhookConfigFileVolume(&deployment.Spec.Template.Spec, auditWebhookRef)
+		deployment.Spec.Template.Spec.TerminationGracePeriodSeconds = ptr.To[int64](130)
```

Comment:
I wonder if we should bump the default value instead.  OpenShift API has a 120 second timeout.

Reply:
Majority of folks are not going to use audit webhook, I would think we'd preferable want to keep the same for them. Then again, increasing TerminationGracePeriodSeconds doesn't really hurt anything. I'm open to that suggestion.

Reply:
Agreed, increasing the grace period is not harmful.  I think we should do it.